### PR TITLE
Fix broken CPack packaging for Apple

### PR DESCRIPTION
**Pull request by reikdas**
CMake attributes like CMAKE_BINARY_DIR are not carried over to CPack scripts.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,4 +32,7 @@ CMakeLists.txt.user
 
 # Other Stuff
 tags
-*.d
\ No newline at end of file
+*.d
+
+# Generated CMake file
+cmake/modules/CPackREADME.cmake
```
Comment (Axel-Naumann) [inline .gitignore:38]: Can you generate this file in the build directory? The source dir should not be touched.

File: cmake/modules/RootCPack.cmake [hunk 2]
```diff
@@ -132,6 +132,7 @@ endif()
 # We want to defer markdown to html conversion; CPACK_INSTALL_SCRIPTS allows this
 # but only takes a .cmake file as argument
 #
+configure_file(${CMAKE_SOURCE_DIR}/cmake/modules/CPackREADME.cmake.in ${CMAKE_SOURCE_DIR}/cmake/modules/CPackREADME.cmake @ONLY)
```
Comment (Axel-Naumann) [inline cmake/modules/RootCPack.cmake:135]: See above: can you generate this file in the build directory? The source dir should not be touched.

**Review by Axel-Naumann — CHANGES_REQUESTED** [review]

File: cmake/modules/RootCPack.cmake (hunk 2 above)
Comment (reikdas) [reply cmake/modules/RootCPack.cmake]: Should be fixed now.

**Review by Axel-Naumann — APPROVED** [review]
Thanks! Given the extra indirections involved (generated cmake file!) I would prefer if we could move the conversion to be part of the build process in a subsequent PR. That way cpack could just pick it up. What do you think?

**Comment by reikdas** [comment]
Sure. I'll try creating a PR for that :)

**Pull request merged**